app/components/channel_icon.js

```diff
@@ -107,13 +105,6 @@ export default class ChannelIcon extends React.PureComponent {
                     </Text>
                 </View>
             );
-        } else if (type === General.DM_CHANNEL && teammateDeletedAt) {
-            icon = (
-                <Image
-                    source={require('assets/images/status/archive_avatar.png')}
```

Comment:
Using archive icon for the deactivated user was odd so I put up offline icon as shown in the screenshot above because in search we do use the offline icon. It is a way to keep it consistent.

Good point as this should probably be bought with UX first

Reply:
have a conversation with UX and update either the code or the description so it is clear what the end result should be